qa/workunits/cephadm/test_cephadm.sh

```diff
@@ -120,6 +120,11 @@ cat <<EOF > $ORIG_CONFIG
 [global]
 	log to file = true
 EOF
+
+$SUDO netstat -anp
+$SUDO docker ps -a || $SUDO podman ps -a
+$SUDO journalctl -u ceph-$FSID@mon.a | cat
```

Comment:
which is basically equal to `cephadm logs --name mon.a | cat`.  I'd be +1 for adding this, not just temporarily!